packages/editor-ui/src/api/annotationTags.ts

```diff
@@ -0,0 +1,22 @@
+import type { IRestApiContext, ITag } from '@/Interface';
+import { makeRestApiRequest } from '@/utils/apiUtils';
+
+export async function getTags(context: IRestApiContext, withUsageCount = false): Promise<ITag[]> {
```

Comment:
Could we use a separate type for the AnnotationTags? Even tho they happen to have the same properties as regular tags, they are a separate entity

Reply:
This file is obsolete, I removed it.
Speaking of separating types: workflow tags and annotation tags share a lot of common logic on the frontend – API, pinia store, tags dropdown, tags container, modal dialog for editing tags, etc. So it will take a lot of work to properly separate them, but do not duplicate lots of code at the same time. I would prefer to make the refactoring after we face some differences between those two types of tags, if this makes sense. What do you think?

Reply:
Makes sense 👍